src/AWS.Deploy.Recipes/DeploymentBundleDefinitions/Container.deploymentbundle

```diff
@@ -11,6 +11,16 @@
             "AdvancedSetting": true,
             "Updatable": true
         },
+        {
+            "Id": "DockerfilePath",
+            "Name": "Dockerfile Path",
+            "Description": "Specifies the absolute path to the Dockerfile as an alternative to the default location.",
+            "Type": "String",
+            "TypeHint": "DockerfilePath",
+            "DefaultValue": "",
+            "AdvancedSetting": true,
+            "Updatable": true
+        },
```

Comment:
You could also add a regex validator to make sure that this is a FilePath. Or create a new validator that checks file paths.

Reply:
IMO,  a new `FilePathExists` validators would suffice since it would verify that the path was not only structurally valid but also exists on disk.

Reply:
Added `FileValidator.cs` to check for existence.